composer.json

```diff
@@ -1,15 +1,44 @@
 {
     "name": "wp-cli/google-sitemap-generator-cli",
+    "description": "WP-CLI commend for google sitemap generator WordPress plugin",
     "license": "MIT",
-    "authors": [{
-        "name": "WP-CLI Team",
-        "homepage": "http://github.com/wp-cli",
-        "role": "Developer"
-    }],
+    "authors": [
+        {
+            "name": "WP-CLI Team",
+            "homepage": "http://github.com/wp-cli",
+            "role": "Developer"
+        }
+    ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.4",
+        "wp-cli/wp-cli": "^2",
+        "wp-cli/wp-cli-tests": "^2.0.7"
```

Comment:
This should move into `require-dev` below.

Reply:
fixed inj https://github.com/wp-cli/google-sitemap-generator-cli/pull/2/commits/8d77acba389ffc674014340122863a1168bb7b1a